src/core/view/View.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Xournal++
+ *
+ * Namespace for view related classes
+ *
+ * @author Xournal++ Team
+ * https://github.com/xournalpp/xournalpp
+ *
+ * @license GNU GPLv2 or later
+ */
+
+#pragma once
+
+#include <gtk/gtk.h>
+
+namespace View {
+
+enum NonAudioTreatment : bool { FADE_OUT_NON_AUDIO_ = true, NORMAL_NON_AUDIO = false };
+enum EditionTreatment : bool { SHOW_CURRENT_EDITING = true, HIDE_CURRENT_EDITING = false };
+enum ColorTreatment : bool { COLORBLIND = true, NORMAL_COLOR = false };
+
+struct Context {
+    cairo_t* cairoContext;
```

Comment:
This will be used everywhere. Should be named `cr`.

Reply:
Done